sql/src/main/java/io/crate/operation/collect/JobCollectContext.java

```diff
@@ -163,7 +171,7 @@ protected void innerStart() {
             if (LOGGER.isTraceEnabled()) {
                 measureCollectTime();
             }
-            collectOperation.launchCollectors(collectors);
+            collectOperation.launchCollectors(collectors, executor);
```

Comment:
would it make sense to just pass the executorName instead of the executor, so that the JobCollectContext doesn't depend on the threadpool?